lib/modules.nix

```diff
@@ -115,8 +115,19 @@ rec {
 
       checkUnmatched =
         if config._module.check && config._module.freeformType == null && merged.unmatchedDefns != [] then
-          let firstDef = head merged.unmatchedDefns;
-          in throw "The option `${showOption (prefix ++ firstDef.prefix)}' defined in `${firstDef.file}' does not exist."
+          let
+            firstDef = head merged.unmatchedDefns;
+            baseMsg = "The option `${showOption (prefix ++ firstDef.prefix)}' defined in `${firstDef.file}' does not exist.";
+          in
+            if attrNames options == [ "_module" ]
+              then throw ''
+                ${baseMsg}
+
+                However it seems as there are no options defined in ${showOption (prefix ++ firstDef.prefix)}. Are you sure you've
```

Comment:
This should be just `showOption prefix` now.

I also still suggest rewording this to `However there are no options defined in ...`, (since that's indeed the case, it doesn't just seem that way :))

Reply:
Oof forgot that, sorry!

Reply:
It's changed now.